bin/phpspec

```diff
@@ -1,26 +1,28 @@
 #!/usr/bin/env php
 <?php
 
-define('PHPSPEC_VERSION', '3.0.x@dev');
```

Comment:
If people need the exact version, there are things like `ocramius/package-versions` instead